README.md

```diff
@@ -39,6 +39,10 @@ app.get('/a-typical-route', async function(req, res, next) {
 app.use(nRaven.errorHandler);
 ```
 
+### Suppressing error logging
+
+By default the raven error handler logs error details as well as sending error information to Sentry. You can suppress this behaviour by setting `response.locals.suppressRavenLogger` to `true` in any route before raven's error handler is registered.
```

Comment:
I am thinking that the middleware has a configuration file. Would it be worth handling the Sentry logging through that configuration file? Otherwise, we have to set the response.locals for all the routes...

Reply:
If we set it in the `n-raven` config then we'll need to also set it in `n-express`, and then every app which uses our new logging middleware will have to manually opt into suppressed Raven errors.

By setting it as a property on the `response` we can make it so that you automatically get this behaviour just by registering `@dotcom-debug-user/middleware-log-errors` before the Raven middleware, you won't need to set it in every route because [it's set by _our_ middleware here](https://github.com/Financial-Times/dotcom-reliability-kit/pull/39/files#diff-463bfc95a3ede98216bd6046470f1c50a8e67e32e30d2c41f212062d28d5353eR60).